resources/keywords/create_provider_account.resource

```diff
@@ -36,55 +41,130 @@ Application Navigate To Create Provider Account Page And Error Message Displayed
+    Element Should Be Visible    ${btnCreateEnabled}
+    Click Element    ${btnCreateEnabled}
+
+User Enters Invalid Data To Create CrunchyDB Provider Account
+    [Arguments]    ${isv}=Crunchy Bridge managed PostgreSQL
+    Set Suite Variable    ${isv}
+    User Enters Provider Account Name And Selects Database Provider    True
+    Wait Until Page Contains Element    ${txtBxCrunchyPubAPI}    timeout=10s
+    Input Text    ${txtBxCrunchyPubAPI}    ${INV_PUB_KEY}
+    Input Text    ${txtBxCrunchyPrivAPI}    ${INV_PRI_KEY}
+    Element Should Be Visible    ${btnCreateEnabled}
+    Click Element    ${btnCreateEnabled}
+
+Provider Account Creation Success
     Application Navigates To Provider Account Creation Success Screen
-    Provider Account Creation Success
+    Provider Account Creation Success Page
+    Provider Account Available on Database Access Screen
 
 Application Navigates To Provider Account Creation Success Screen
     Wait Until Keyword Succeeds     5x     2s      Wait Until Page Contains Element    ${lblDBInstSuccess}
     Element Should Be Visible    ${btnViewProviderAcc}
 
-Provider Account Creation Success
+Provider Account Creation Success Page
     Click Element    ${btnViewProviderAcc}
-    Wait Until Page Contains Element    ${btnLastUpdatedHeader}     timeout=10s
+    Wait Until Page Contains Element    ${titleDbaasInvertories}    timeout=10s
     Filter ${provaccname} On DBaaS Inventory
     ${elemDbaasInvStatusUpd}=    Replace String    ${elemDbaasInvStatus}    <<paname>>    ${provaccname}
     Wait Until Keyword Succeeds     10x     2s      Wait Until Page Contains Element    ${elemDbaasInvStatusUpd}
     Set Suite Variable    ${provaccname}
 
-User Creates MongoDB Provider Account
+Provider Account Available on Database Access Screen
+    User Navigates To Database Access Under Data Services
+    Wait Until Page Contains Element    ${titleDatabaseAccess}
+    ${elemProvAccUpd}=    Replace String    ${elemProvAcc}    <<paname>>    ${provaccname}
+    ${updProvAcc}=    Replace String    ${elemDBProvider}    <<paname>>    ${provaccname}
+    ${isvName}=    Fetch From Left    ${isv}    ${SPACE}
+    Log To Console    ${isvName}
+    ${elemDBProviderUpd}=    Replace String    ${updProvAcc}    <<isvname>>    ${isvName}
+    Element Should Be Visible    ${elemProvAccUpd}
+    Element Should Be Visible    ${elemDBProviderUpd}
+
+Provider Account Creation Failure
+    Provider Account Creation Failure Screen
+    User Navigates To DBaaS Inventories Screen
+    Check Status Of Invalid Provider Account
+
+Provider Account Creation Failure Screen
+    Wait Until Page Contains Element    ${titleCreateProviderAcc}    timeout=15s
+    Wait Until Page Contains Element     ${lblDBInstFailure}    timeout=20s
+    Page Should Contain Button    ${btnEditProvAcc}
+
+User Navigates To DBaaS Inventories Screen
+    User Navigates To Installed Operators Under Operators
+    User Filters redhat-dbaas-operator Namespace On Project Dropdown
```

Comment:
Shouldn't we use ${PROJECT} like before?

Reply:
Parameterized Project Namespace using variable _${PROJECT}_ with default value _redhat-dbaas-operator_